packages/types/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@instructure/types",
+  "version": "8.7.0",
+  "description": "Shared TypeScript typings for Instructure UI packages.",
+  "author": "Instructure, Inc. Engineering and Product Design",
+  "homepage": "https://instructure.design/#types",
+  "license": "MIT",
+  "main": "src/index.ts",
+  "types": "dist/index.d.ts",
+  "private": true,
```

Comment:
As we've discussed this could be private, but we should state in the Readme that they should not use types that are for internal APIs because they might change.

Reply:
I've changed this to be public and will explain in the package's readme that it should be used with caution because of the internal usage.